libs/api-bot/src/Network/Wire/Bot/Monad.hs

```diff
@@ -137,24 +141,31 @@ data BotNetEnv = BotNetEnv
     }
 
 newBotNetEnv :: Manager -> Logger -> BotNetSettings -> IO BotNetEnv
-newBotNetEnv m l o = do
+newBotNetEnv manager logger o = do
     gen <- MWC.createSystemRandom
-    usr <- maybe Cache.empty (Cache.new l gen) (setBotNetUsersFile o)
+    usr <- maybe Cache.empty (Cache.fromFile logger gen) (setBotNetUsersFile o)
     mbx <- maybe (return []) loadMailboxConfig (setBotNetMailboxConfig o)
     met <- initMetrics
-    let sdr = setBotNetSender o
     let srv = Server
             { serverHost    = setBotNetApiHost   o
             , serverPort    = setBotNetApiPort   o
             , serverWsHost  = setBotNetApiWsHost o
             , serverWsPort  = setBotNetApiWsPort o
             , serverSSL     = setBotNetApiSSL    o
-            , serverManager = m
+            , serverManager = manager
             }
-    let asrt = setBotNetAssert o
-    let sets = setBotNetBotSettings o
-    let rprt = setBotNetReportDir o
-    return $! BotNetEnv gen mbx sdr usr srv l asrt sets met rprt
```

Comment:
I expanded the record constructor because in general I think that using the "lightweight" syntax for bigger records is a poor habit, especially if said records contain any `Bool`s or `Maybe`s